### ISSUE TITLE
negative hexadecimal numbers

### ISSUE THREAD

**Issue by jorisbontje** [ethereum/ethereum.js#97]
`web3.fromDecimal(-1)` returns `0x-1`, this should be `-0x1` instead. At least that is the format supported by JavaScript, Python, Ruby, etc.

**Comment by chevdor** [comment]
confirmed

Added a unit test:
```
describe('web3', function(){
    it('should convert fromDecimal(-1) into -0x1', function(){
        assert.equal(web3.fromDecimal(-1), '-0x1');
    });
});
````

**Comment by chevdor** [comment]
Hi @jorisbontje,
Thanks for the report. I have fixed the issue and added a few tests.
The fix is in the pull request #103

**Comment by debris** [comment]
fixed in 9fe105e7946bcd59ef9a88b6ad8303b6eaa26346, will merge do develop soon
